src/unix.rs

```diff
@@ -161,39 +158,38 @@ pub type ChildStdin = PollEvented<Fd<process::ChildStdin>>;
 pub type ChildStdout = PollEvented<Fd<process::ChildStdout>>;
 pub type ChildStderr = PollEvented<Fd<process::ChildStderr>>;
 
-impl<T> Evented for Fd<T> where T: AsRawFd {
-    fn register(&self,
-                poll: &mio::Poll,
-                token: Token,
-                interest: Ready,
-                opts: PollOpt)
-                -> io::Result<()> {
-        EventedFd(&self.0.as_raw_fd()).register(poll,
-                                                token,
-                                                interest | UnixReady::hup(),
-                                                opts)
-    }
-
-    fn reregister(&self,
-                  poll: &mio::Poll,
-                  token: Token,
-                  interest: Ready,
-                  opts: PollOpt)
-                  -> io::Result<()> {
-        EventedFd(&self.0.as_raw_fd()).reregister(poll,
-                                                  token,
-                                                  interest | UnixReady::hup(),
-                                                  opts)
+impl<T> Evented for Fd<T>
+where
```

Comment:
Personally prefer that were clauses are indented once (better readability IMO)

Reply:
This is configurable as well so I'll change: https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#where-predicates